onfido-java/src/main/models/task.json

```diff
@@ -0,0 +1,44 @@
+{
+  "title": "Task",
+  "properties": {
+    "created_at": {
+      "type": "string",
+      "format": "date-time",
+      "description": "The date and time at which the task was executed",
+      "readOnly": true
+    },
+    "id": {
+      "type": "string",
+      "description": "The unique identifier for the task",
+      "readOnly": true
+    },
+    "task_def_id": {
+      "type": "string",
+      "description": "The unique identifier for the task"
+    },
+    "task_def_version": {
+      "type": "string",
+      "description": "The task version"
+    },
+    "updated_at": {
+      "type": "string",
+      "format": "date-time",
+      "description": "The date and time at which the task was last updated",
+      "readOnly": true
+    },
+    "workflow_run_id": {
+      "type": "string",
+      "description": "The unique identifier for the workflow run"
+    },
+    "input": {
+      "$ref": "task_input.yaml",
+      "readOnly": true,
+      "description": "Data provided for this task to be executed"
+    },
+    "output": {
```

Comment:
Both attributes, input and output are dynamic objects so this has to be treated as such. You can see an example in [here](https://github.com/onfido/onfido-java/blob/master/onfido-java/src/main/models/workflow_run.json#L56).